examples/playground/index.html

```diff
@@ -173,5 +173,37 @@
 
       <a-entity light="type: point; color: #f4f4f4; intensity: 0.4; distance: 0" position="0 2 0"></a-entity>
     </a-scene>
+
+    <script>
+      // Add the drag and drop event listeners to the document and process a dragged GLB file.
+      document.addEventListener("dragover", function(e) {
+          e.stopPropagation();
+          e.preventDefault();
+          e.dataTransfer.dropEffect = "copy";
+      }, false);
+
+      document.addEventListener("drop", function(e) {
+          e.stopPropagation();
+          e.preventDefault();
+
+          const files = e.dataTransfer.files; // Array of all files.
+
+          for (let i = 0, file; file = files[i]; i++) {
```

Comment:
I prefer the more modern for-of syntax. Can you please try
` for (const file of files) {`